### PR TITLE
Refactoring code

### PR DESCRIPTION
**Pull request by barakadanny**
In this PR I refactor the code following:

1. The SOLID principale

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ To get a local copy up and running follow these simple example steps.
 - GitHub: [@barakadan](https://github.com/barakadanny)
 - LinkedIn: [danny baraka](https://www.linkedin.com/in/danny-baraka-589156169/)
 
+👤 **George William Kidde**
+  - Github: [@kidde60](https://github.com/kidde60)
+  - Twitter: [@KGWilliam60](https://twitter.com/KGWilliam60)
+  - Linkedin: [Linkedin](https://www.linkedin.com/in/kidde-george-william/)
 
 ## 🤝 Contributing
```
Comment (youmari) [inline README.md:36]: - [x] note I'm reviewing here cause the file doesn't available here 

- [x] Your code is clean however you should separate the code that asks for the user's answer and the one that validates if it's  `yes or no or 1 or 2 ....etc` into private methods and used them in your methods to create the book, person, ...etc in order to achieve the `single responsibility principle` please make sure to do that in all the methods 
Screenshot 2022-11-21 at 14 20 15

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @barakadanny , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Great commit messages
✔️ PRO PR
✔️ No linter error 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb [hunk 2]
```diff
@@ -51,24 +51,30 @@ def create_person
     end
   end
```
Comment (HAMMAS-SALEEM) [inline app.rb:53]: image

- [x] Its great that you have implemented the requested changes but i guess you have broken your application while refactoring your application. As you can see in the image above :arrow_up: while creating a rental your application shows an error :fearful: Kindly fix that :+1:

**Review by HAMMAS-SALEEM — CHANGES_REQUESTED** [review]
# Required Changes :recycle:
almost-there

## Hey @barakadanny :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Title
✔️ Descriptive PR Summary
✔️ Following Gitflow
✔️ Following some requirements

Good job on implementing some of the reqirements, there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ShadyShawkat — APPROVED** [review]
## Hi @barakadanny, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All linters are passing ✔️ 
- Correct use of GitFlow ✔️ 
- Descriptive commit messages ✔️ 
- Professional looking pull request ✔️ 
- Your readme looks professional ✔️ 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**